File: Python/1.py
# Two Sum
from typing import List

class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        return_list: List[int] = []

        for i in range(len(nums) - 1):
            for j in range(i + 1, len(nums)):
                if target - nums[j] == nums[i]:
                    return_list.append(i)
                    return_list.append(j)
                    return return_list

        return return_list

File: JS/1.js
// Two Sum

var twoSum = function(nums, target) {
    let arrayNums = [];

    for(let i = 0; i < nums.length - 1; i++){
        for(let j = i + 1; j < nums.length; j++ ){
            if ( target - nums[j] == nums[i]){
                arrayNums[0] = i;
                arrayNums[1] = j;
                return arrayNums;
            }
        }
    }

    return arrayNums;
};